### PR TITLE
ChatQnA on Gaudi - fix tabs for build/pull, use TAG instead of specific version

### PR DESCRIPTION
**Pull request by alexsin368**
Fixing the formatting of the tabs to hide the build sections when "pull" is selected.
Use TAG instead of the specific version number for maintainability.

### CODE REVIEW

**Review by mkbhanda — APPROVED** [review]
LGTM

File: examples/ChatQnA/deploy/gaudi.md [hunk 1]
```diff
@@ -139,20 +142,20 @@ docker build --no-cache -t opea/reranking-tei:latest --build-arg https_proxy=$ht
 
 Build vLLM docker image with hpu support
 ```
-docker build --no-cache -t opea/llm-vllm-hpu:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu .
+docker build --no-cache -t opea/llm-vllm-hpu:${TAG} --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu .
```
Comment (ashahba) [inline examples/ChatQnA/deploy/gaudi.md:145]: I'm getting:
```
ls: cannot access 'comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_hpu': No such file or directory
```
and the only Dockerfile I see in that location is:
`comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_gpu`

Comment (ashahba) [reply examples/ChatQnA/deploy/gaudi.md:145]: I think that means, we need:
```
docker build --no-cache -t opea/llm-vllm-gpu:${TAG} --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/vllm/langchain/dependency/Dockerfile.intel_gpu .
```
or something like that in the README.

Comment (alexsin368) [reply examples/ChatQnA/deploy/gaudi.md]: Need to build for Gaudi/HPU, so using this step instead:

bash ./build_docker_vllm.sh hpu

**Pull request merged**